downloadPdfCustomSelectionToolbar/downloadPdfCustomSelectionToolbar.js

```diff
@@ -43,6 +45,13 @@ var updateDownloadPdfCustomSelectionToolbarSelectButtonLabel = function(event) {
     }
 };
 
+var updateDownloadPdfCustomSelectionIndicators = function (event) {
+    if (event.data.leftPageNumber == null || event.data.rightPageNumber == null || event.data.leftPageNumber === 0) {
+        downloadPdfCustomSelectionCheckBoxes.left.css('visibility', 'hidden');
```

Comment:
If it's the right page number that's missing, shouldn't the right checkbox be hidden?